src/applications/app.helpers.js

```diff
@@ -8,6 +8,7 @@ export const BOOTSTRAPPING = 'BOOTSTRAPPING';
 export const NOT_MOUNTED = 'NOT_MOUNTED';
 export const MOUNTING = 'MOUNTING';
 export const MOUNTED = 'MOUNTED';
+export const UPDATING = 'UPDATING';
```

Comment:
A new status for when parcels are updating

Reply:
Should this live in a new file for parcel helpers? The UPDATING status doesn't apply to applications.

Reply:
the other statuses in this file don't only apply to applications, either. I think we could move the statuses into a separate file inside of the lifecycles directory, but didn't want to change everything in one pr and blow this up into a bigger and bigger refactor, so I didn't.